config/locales/en.yml

```diff
@@ -2311,6 +2311,7 @@ en:
   label_product_version: "Product version"
   label_profile: "Profile"
   label_percent_complete: "% Complete"
+  label_project: "Project"
```

Comment:
I see the translation key was added, so why would the default be required/what does it bring with it?

Reply:
Answered for app/models/queries/projects/filters/id_filter.rb